lib/api/openid_connect/id_token.rb

```diff
@@ -25,15 +25,15 @@
 
 module Api
   module OpenidConnect
-    class IdToken < ActiveRecord::Base
-      belongs_to :authorization
+    class IdToken
+      attr_accessor :authorization, :expires_at, :created_at, :nonce
```

Comment:
`attr_reader`s should be enough, right? If we need to expose any at all that is.